frontend/src/locales/en/translation.json

```diff
@@ -976,26 +991,27 @@
         "image": "Image",
         "richText": "Rich Text",
         "timer": "Timer",
-        "iframe": "iframe",
+        "iframe": "iFrame",
```

Comment:
From end user point, this world may be just `frame` is more reasonable @xieliuduo @scottsut , but it is ok for this commit. If we decide to use  a more "reasonable" world, we would try to fix it.